go/vt/vtctl/grpcvtctldserver/server.go

```diff
@@ -2432,6 +2434,44 @@ func (s *VtctldServer) SetKeyspaceServedFrom(ctx context.Context, req *vtctldata
 	}, nil
 }
 
+// SetKeyspaceDurabilityPolicy is part of the vtctlservicepb.VtctldServer interface.
+func (s *VtctldServer) SetKeyspaceDurabilityPolicy(ctx context.Context, req *vtctldatapb.SetKeyspaceDurabilityPolicyRequest) (*vtctldatapb.SetKeyspaceDurabilityPolicyResponse, error) {
```

Comment:
alphabetically, this should come before `SetKeyspaceServedFrom`

Reply:
fixed